src/typechecker/FStar.TypeChecker.Util.fsi

```diff
@@ -76,7 +76,9 @@ val check_trivial_precondition : env -> comp -> (comp_typ * formula * guard_t)
 //checking that e:t is convertible to t'
 val check_and_ascribe : env -> term -> typ -> typ -> term * guard_t
 val check_top_level: env -> guard_t -> lcomp -> bool*comp
-val maybe_coerce_bool_to_type: env -> term -> lcomp -> typ -> term * lcomp
+val maybe_coerce_lc : env -> term -> lcomp -> typ -> term * lcomp
+val maybe_coerce    : env -> term -> typ   -> typ -> term * typ
```

Comment:
Minor: does `maybe_coerce` need to be in the interface? It seems like an internal function (but don't really mind).